src/tests/admin/conferenceRoomResourceAssociations/AResourceDeletedIsRemovedFromAvailablesGridInARoom.java

```diff
@@ -0,0 +1,77 @@
+package tests.admin.conferenceRoomResourceAssociations;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import jxl.read.biff.BiffException;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import framework.common.UIMethods;
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomInfoPage;
+import framework.pages.admin.conferencerooms.RoomResourceAssociationsPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.pages.admin.resources.ResourcesPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+import framework.utils.readers.JsonReader;
+
+/**
+ * TC04: Verify that a deleted resource is removed from {Availables} grid in a room 
+ * @author Juan Carlos Guevara 
+ */
+public class AResourceDeletedIsRemovedFromAvailablesGridInARoom {
```

Comment:
I think you should change the ResourceDeleted to DeletedResource in the name of the class. The same in the name of the test method